golang/lib/networks/network_context.go

```diff
@@ -409,3 +435,35 @@ func (networkCtx *NetworkContext) ExecuteBulkCommands(bulkCommandsJson string) e
 	}
 	return nil
 }
+
+// Docs available at https://docs.kurtosistech.com/kurtosis-client/lib-documentation
+func (networkCtx *NetworkContext) GetServices() ([]services.ServiceID, error){
```

Comment:
there can only be one of each service ID in teh network, so we actaully want the return type to be a set. Go doesn't have sets, so we should use `map[services.ServiceID]bool` instead (this is the official recommendation :| )